stacker/actions/build.py

```diff
@@ -90,8 +90,6 @@ def _build_stack_tags(self, stack, template_url):
             'template_url': template_url,
```

Comment:
Lets go ahead and remove template_url as well, I don't believe it's used anywhere, and it ends up being misleading.